com.simplenote.Simplenote.yaml

```diff
@@ -1,8 +1,8 @@
 app-id: com.simplenote.Simplenote
 base: org.electronjs.Electron2.BaseApp
-base-version: '20.08'
+base-version: '21.08'
```

Comment:
22.08 is out.

Reply:
It has been explicitly stated in the comment that the application does not launch with 22.08 and I will look into a fix at a later date.